new-lamassu-admin/src/routing/routes.js

```diff
@@ -13,11 +13,14 @@ import {
 } from 'react-router-dom'
 
 import { AppContext } from 'src/App'
-import AuthRegister from 'src/pages/AuthRegister'
+import Login from 'src/pages/Authentication/Login'
+import Register from 'src/pages/Authentication/Register'
+import Reset2FA from 'src/pages/Authentication/Reset2FA'
+import ResetPassword from 'src/pages/Authentication/ResetPassword'
 import Blacklist from 'src/pages/Blacklist'
 import Cashout from 'src/pages/Cashout'
 import Commissions from 'src/pages/Commissions'
-import ConfigMigration from 'src/pages/ConfigMigration'
+// import ConfigMigration from 'src/pages/ConfigMigration'
```

Comment:
remove comments

Reply:
It most likely is a result of the rebase, but I'll remove it